packages/shell-api/src/shell-instance-state.ts

```diff
@@ -496,6 +496,8 @@ export default class ShellInstanceState {
     const extraConnectionInfo = this.connectionInfo?.extraInfo;
     if (extraConnectionInfo?.is_data_federation) {
       return 'AtlasDataFederation';
+    } else if (extraConnectionInfo?.is_local_atlas) {
+      return 'Atlas Local Dev';
```

Comment:
This sounds okay to me, but in case you haven’t, feel free to confirm the exact phrase here with @mmarcon :slightly_smiling_face: 

It might be worth being consistent between ADF and this and either use spaces in both of them or in neither?

Reply:
add changed it to `LocalDev` and pinged Max on same. If he suggests to change it, I'll do that accordingly.

Reply:
Update from Max - he suggested to use `AtlasLocalDev` and I added that in [57f84af](https://github.com/mongodb-js/mongosh/pull/1629/commits/57f84af45a567f86ea020b1cf1caaf0fd26069e8)